Add async_scanner_devices_by_address to Bluetooth (#1589)
home-assistant/developers.home-assistant: docs/bluetooth.md

service_infos = bluetooth.async_discovered_service_info(hass, connectable=True)
```

### Fetching all discovered devices and advertisment data by each bluetooth adapter

> To access the list of previous discoveries and advertisment data received by each adapter independently, call the `bluetooth.async_get_discovered_devices_and_advertisement_data_by_address` API. The same device and advertisment data may appear multiple times, once per bluetooth adapter that reached it.

Reviewer: ```suggestion
To access the list of previous discoveries and advertisment data received by each adapter independently, call the `bluetooth. async_scanner_devices_by_address` API. The same device and advertisment data may appear multiple times, once per bluetooth adapter that reached it.
```

```python
from homeassistant.components import bluetooth

> discovered_devices_and_advertisement_data = bluetooth.async_get_discovered_devices_and_advertisement_data_by_address(hass, "44:44:33:11:23:42", connectable=True)

Reviewer: ```suggestion
devices = bluetooth. async_scanner_devices_by_address(hass, "44:44:33:11:23:42", connectable=True)
```

Author: A rogue space slipped through: `device = bluetooth.-->[ ]<--async_scanner_devices_by_address`

Reviewer: Good catch. Fixed suggestion.

service_infos = bluetooth.async_discovered_service_info(hass, connectable=True)
```

> ### Fetching all discovered devices and advertisment data by each bluetooth adapter

Reviewer: ```suggestion
### Fetching all discovered devices and advertisement data by each Bluetooth adapter
```

> To access the list of previous discoveries and advertisment data received by each adapter independently, call the `bluetooth.async_scanner_devices_by_address` API. The call returns a list of `BluetoothScannerDevice` objects. The same device and advertisment data may appear multiple times, once per bluetooth adapter that reached it.

Reviewer: ```suggestion
To access the list of previous discoveries and advertisement data received by each adapter independently, call the `bluetooth.async_scanner_devices_by_address` API. The call returns a list of `BluetoothScannerDevice` objects. The same device and advertisement data may appear multiple times, once per Bluetooth adapter that reached it.
```

```python
from homeassistant.components import bluetooth

device = bluetooth.async_scanner_devices_by_address(hass, "44:44:33:11:23:42", connectable=True)
# device.ble_device is a bleak `BLEDevice`

> # device.advertisement is a bleak `AdvertisementData`

Reviewer: ```suggestion
# device.advertisement is a bleak `AdvertisementData`
```
